blocks/library/embed/index.js

```diff
@@ -76,6 +81,7 @@ registerBlockType( 'core/embed', {
 		constructor() {
 			super( ...arguments );
 			this.doServerSideRender = this.doServerSideRender.bind( this );
+			this.isPreviewBlacklisted = this.isPreviewBlacklisted.bind( this );
```

Comment:
Technically this function may not need to be bound because it's never called in a context that would have a `this` binding anything other than the current component instance (usually callbacks). But I think it's probably wise to err on the side of safety and `bind` every component function.